### PR TITLE
Adding FAQs

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: FAQs/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+
+</br>
+
+# FAQs
+
+- ### Setup
+ 1. [Getting error "Class keyword is not allowed in ConstrainedLanguage mode"](#1getting-error--class-keyword-is-not-allowed-in-constrainedlanguage-mode)
```
Comment (asimks) [inline FAQs/README.md:9]: Links are not working currently. Kindly check them once merged to main branch to confirm if they are working fine.

File: FAQs/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+ 1. [I am getting alert mail "AzTS MONITORING ALERT: AzTS Auto-Updater Failure Alert". What does it mean? How to stop/resolve this alert](#1-i-am-getting-alert-mail-azts-monitoring-alert-azts-auto-updater-failure-alert-what-does-it-mean-how-to-stopresolve-this-alert)
+
+--------------------------------------------------
+</br>
+
+## **FAQs **
+- ### Setup
+
+ ### **1. Getting error : Class keyword is not allowed in ConstrainedLanguage mode** 
+This error can be observed where PowerShell is running in ConstrainedLanguage mode. If local settings for LaguageMode cannot be modified for some reason, AzTa setup can be done using aletrnate options. One of the options be copying deployment files to Azure storage and running the whole deployment procedure within Cloud Shell.
```
Comment (asimks) [inline FAQs/README.md:29]: AzTa  -  Typo

Comment (asimks) [inline FAQs/README.md:29]: One of the **options be** copying deployment files -> One of the **option is** copying deployment files

File: FAQs/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+ 
+2.	If there are no exceptions, then please check  traces of the Job in Application Insight (AzSK-AzTS-AppInsights-xxxxx) by running following query:
+traces
+| where timestamp > ago(3d)
+| where operation_Name =~ "AutoUpdater"
+| order by timestamp desc
+| project timestamp, message, severityLevel
+
+If you see traces "Exception occurred in UpdateFunctionApp function while updating [AzSK-AzTS-UI-xxxxx]. Exception : [System.Exception: App service [AzSK-AzTS-UI-xxxxx] request to swap slot with production failed", then Auto-Updater is failing as staging and production slot swapping is failing for AzTS UI.You can disable the slot swap feature in Auto Updater with below steps:
+1. Go to Auto Updater Function App (AzSK-AzTS-AutoUpdater-b93ac)
```
Comment (asimks) [inline FAQs/README.md:159]: AzSK-AzTS-AutoUpdater-b93ac -> AzSK-AzTS-AutoUpdater-xxxxx

File: 01-Setup and getting started/README.md [hunk 4]
```diff
@@ -57,6 +57,11 @@ Let's start!
   If the PSVersion is older than 5.0, update PowerShell from [here](https://www.microsoft.com/en-us/download/details.aspx?id=54616).  
 
       ![PowerShell Version](../Images/00_PS_Version.png)
+    
+ 1. c.  PowerShell language mode FullLanguage: </br>
```
Comment (TarunKrShukla) [inline 01-Setup and getting started/README.md:61]: Somehow if you to go file view, this comes as '2.c' instead of '1.c'

Comment (TarunKrShukla) [reply 01-Setup and getting started/README.md:61]: We need to add this text in 'method B' as well.

File: FAQs/README.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+ - ### Scan
+ 1. [Today's AzTS scan has completed. How do I re-run the full scan?](#1-todays-azts-scan-has-completed-how-do-i-re-run-the-full-scan)
+ 2. [How to disable AzTS scan and uninstall the setup?](#2-how-to-disable-azts-scan-and-uninstall-the-setup)
+ 3. [The subscription scan in AzTS is getting terminated due to function timeout. How can I fix it? OR How can I upgrade the pricing tier of AzTS function apps?](#3-the-subscription-scan-in-azts-is-getting-terminated-due-to-function-timeout-how-can-i-fix-it-or-how-can-i-upgrade-the-pricing-tier-of-azts-function-apps)
+
+- ### Monitoring
+ 1. [I am getting alert mail "AzTS MONITORING ALERT: AzTS Auto-Updater Failure Alert". What does it mean? How to stop/resolve this alert](#1-i-am-getting-alert-mail-azts-monitoring-alert-azts-auto-updater-failure-alert-what-does-it-mean-how-to-stopresolve-this-alert)
```
Comment (TarunKrShukla) [inline FAQs/README.md:20]: Typo: '?' missing

File: FAQs/README.md (hunk 2 above)
Comment (Aboli-msft) [reply FAQs/README.md]: Fixed, thanks!!

File: 01-Setup and getting started/README.md (hunk 4 above)
Comment (Aboli-msft) [reply 01-Setup and getting started/README.md]: That was weird, some indentation related issue. Fixed, Thanks for the catch !

File: FAQs/README.md (hunk 5 above)
Comment (Aboli-msft) [reply FAQs/README.md]: Fixed, thanks!!

File: FAQs/README.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+
+</br>
+
+# FAQs
+
+- ### Setup
+ 1. [Getting error "Class keyword is not allowed in ConstrainedLanguage mode"](#1-getting-error--class-keyword-is-not-allowed-in-constrainedlanguage-mode)
+ 2. [How to add new subscriptions or mangement groups after deploying AzTS?](#2-how-to-add-new-subscriptions-or-management-groups-after-deploying-azts)
+ 3. [On running the AzTS installation command (`Install-AzSKTenantSecuritySolution`) I am getting an error message *"Tenant ID, application ID, principal ID, and scope are not allowed to be updated."](#3-on-running-the-azts-installation-command-install-azsktenantsecuritysolution-i-am-getting-an-error-message-tenant-id-application-id-principal-id-and-scope-are-not-allowed-to-be-updated)
```
Comment (asimks) [inline FAQs/README.md:11]: * before the "

File: FAQs/README.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+ 2. [How to disable AzTS scan and uninstall the setup?](#2-how-to-disable-azts-scan-and-uninstall-the-setup)
+ 3. [The subscription scan in AzTS is getting terminated due to function timeout. How can I fix it? OR How can I upgrade the pricing tier of AzTS function apps?](#3-the-subscription-scan-in-azts-is-getting-terminated-due-to-function-timeout-how-can-i-fix-it-or-how-can-i-upgrade-the-pricing-tier-of-azts-function-apps)
+
+- ### Monitoring
+ 1. [I am getting alert mail "AzTS MONITORING ALERT: AzTS Auto-Updater Failure Alert". What does it mean? How to stop/resolve this alert?](#1-i-am-getting-alert-mail-azts-monitoring-alert-azts-auto-updater-failure-alert-what-does-it-mean-how-to-stopresolve-this-alert)
+
+--------------------------------------------------
+</br>
+
+## **FAQs **
```
Comment (asimks) [inline FAQs/README.md:25]: ** is coming around the FAQ

Comment (TarunKrShukla) [inline FAQs/README.md:25]: ** can be removed.

File: FAQs/README.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+    ```
+
+4. To increase function timeout, go to your function app (say, you want to increase timeout value for 'AzSK-AzTS-WorkItemProcessor-xxxxx'. This app contains function to scan subscription with baseline control.) > Settings > Configuration > Application settings > Update the value of `AzureFunctionsJobHost__functionTimeout` to '01:00:00' to increase the timeout value to 1 hour.
+
+  > _**Note:** In future if you run the AzTS installation command (`Install-AzSKTenantSecuritySolution`) to upgrade your existing AzTS setup, you will have to repeat the above steps._
+<br> 
+<br>
+
+
+- ### Monitoring
```
Comment (TarunKrShukla) [inline FAQs/README.md:139]: Instead of maing bullet list, see if just sub heading wih ** ** to make text bold looks better.

**Review by TarunKrShukla — APPROVED** [review]

**Review by asimks — APPROVED** [review]

**Pull request merged**